src/gt4py/next/program_processors/codegens/gtfn/codegen.py

```diff
@@ -269,6 +269,8 @@ class GTFNIMCodegen(GTFNCodegen):
 
     InitStmt = as_fmt("{init_type} {lhs} {op} {rhs};")
 
+    EmptyListInitializer = as_fmt("{{}}")
```

Comment:
```suggestion
    EmptyListInitializer = as_mako("{}")
```
nit, just saving the escape characters